cmd/server/main.go

```diff
@@ -26,7 +26,7 @@ import (
 
 const databaseURITemplate = "host=%s port=%d dbname=%s user=%s password=%s sslmode=%s"
 
-var config = flag.String("config", "config.yaml", "Path to the config file for the server")
+var config = flag.String("config", "./", "Path to the config file for the server")
```

Comment:
This is a bit different than my intended use of this flag but your change makes sense so I'm for trying it. Just one nit to update the flag help text though:

```suggestion
var config = flag.String("config", "./", "Path to the directory containing the server config file")
```

Reply:
Yeah you're right thanks